CMakeLists.txt

```diff
@@ -110,8 +110,8 @@ dawn_report_result("Package summary" ${package_info})
 # Add clang-format target
 set(format_directories 
   "${CMAKE_SOURCE_DIR}/src"
-  "${CMAKE_SOURCE_DIR}/test/integration-test"
   "${CMAKE_SOURCE_DIR}/test/unit-test"
+#  "${CMAKE_SOURCE_DIR}/test/integration-test"

```

Comment:
should we disable this? arent they working anymore?

Reply:
yes, the problem is that clang-format gets confused with gtclang syntax (vertical regions are a difficult thing and we use custom keywords that are not recognized. This is bad since the integrationtests for the serialization of the SIR are sensitive to the position of specific arguments. Therefore I think this should be disabled

Reply:
ah ok, sorry I thought you were disabling the test, but you are disabling the formatting. Fine then. just remove the comment